### PR TITLE
remove default approvals count 1

### PR DESCRIPTION
**Pull request by shine2lay**
based on feed backs (some complaint about having default approval count 1), it is removed in this PR

### CODE REVIEW

**Comment by jusx** [comment]
@shine2lay we need to write unit test that if approvals is not defined that the validator does not crash.

**Comment by shine2lay** [comment]
@jusx test added

File: lib/approvals.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const _ = require('lodash')
  *  `mergeable` is false
  */
 module.exports = async (pr, context, settings) => {
-  let approvalSettings = settings.mergeable.approvals
+  let approvalSettings = settings.mergeable.approvals ? settings.mergeable.approvals : 0
```
Comment (jusx) [inline lib/approvals.js]: Just a suggestion. But I think this can just be written as `let approvalSettings = settings.mergeable.approvals || 0`

Comment (shine2lay) [reply lib/approvals.js]: you're right

**Pull request merged**